content/javascript/concepts/map/terms/delete/delete.md

```diff
@@ -37,3 +37,33 @@ console.log(fruits.delete('Strawberries')); // Output: false
 ```
 
 The first `.delete()` statement returns `true` because `fruits` contain an entry with `Oranges` as a key and has successfully removed it. The second statement returns `false` because an entry with the `Strawberries` key does not exist in `fruits`.
+
+## Codebyte Example
+
+In the example below, we declare a new inventory `Map` of consumable construction supplies linking them to their current quantities on hand. Throughout this example:
```

Comment:
```suggestion
In the code below, a new inventory `Map` of consumable construction supplies catalogs the current quantities on hand. In this example:
```

Reply:
generally we avoid the use of first and second person pronouns in docs (I, you, we)

Reply:
I didn't even consider that! Great catch.